lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -592,10 +592,10 @@ public void flushIndexing(final ObjectTypeParsingRules rule) throws IOException
                 Collectors.toMap(Function.identity(), guid -> map.containsKey(guid))));
     }
 
-    private Integer loadLastVersion(String reqIndexName, GUID parentGUID, 
+    private Integer loadLastVersion(GUID parentGUID,
             Integer processedVersion) throws IOException {
 
-        reqIndexName = getGeneralIndexName(reqIndexName);
+        final String reqIndexName = "*";
```

Comment:
You don't want to do this as it will search outside the namespace.

Reply:
Isn't this what we want? How else would you know what is the last version of something?

Reply:
By namespace, do you mean the index name? I am not sure what you mean by application namespace

Reply:
Resolved this offline